lib/datasource/nuget/index.spec.ts

```diff
@@ -404,19 +409,30 @@ describe('datasource/nuget', () => {
       httpMock
         .scope('https://api.nuget.org')
         .get('/v3/registration5-gz-semver2/nunit/index.json')
-        .reply(200, pkgListV3Registration);
+        .reply(
+          200,
+          pkgListV3Registration
+            .replace(/"http:\/\/nunit\.org"/g, '""')
+            .replace('"published": "2012-10-23T15:37:48+00:00",', '')
+        )
+        .get('/v3-flatcontainer/nunit/3.12.0/nunit.nuspec')
+        .reply(
+          200,
+          pkgInfoV3FromNuget.replace('https://github.com/nunit/nunit', '')
+        );
```

Comment:
Is it worthwhile to have a test checking that `sourUrl` is only set once? i.e. return pkgInfo with 2 different homepages, and make sure that only the first one is used?

Reply:
This is just to make coverage happy. 😇

Reply:
Okay, no worries.